net/kcptun/Makefile

```
@@ -1,13 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=kcptun
-PKG_VERSION:=20210922
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=20230207
+PKG_RELEASE:=2
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/xtaci/kcptun/tar.gz/v${PKG_VERSION}?
-PKG_SOURCE_DATE:=2021-09-22
-PKG_HASH:=f6a08f0fe75fa85d15f9c0c28182c69a5ad909229b4c230a8cbe38f91ba2d038
+PKG_SOURCE:=v20230207.tar.gz
+PKG_SOURCE_URL:=https://github.com/xtaci/kcptun/archive/refs/tags/
```

```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/xtaci/kcptun/archive/refs/tags/v$(PKG_VERSION).tar.gz?
```

So, this is what I actually meant.
See http://sources.openwrt.org/, only God knows what those version.tar.gz files are.

---

@1715173329  I agree with u. 
but the name comes from the repo of [PKG_SOURC_URL](https://github.com/xtaci/kcptun/archive/refs/tags/), I can't find the latest version in the repo of http://sources.openwrt.org/

---

sources is the mirror, all source files downloaded by the openwrt buildbot will be uploaded there.
Of course, this commit is not applied yet, so new tarball doesn't exist on it.

See my suggestion, use `?`in URL  to override the original file name.

And codeload still works for me, could you check your network issue etc.?

---

@1715173329
I have changed the [PKG_SOURC_URL](https://github.com/xtaci/kcptun/archive/refs/tags/) to origin URL. the reason why I changed was that I can't download the source in my SKD environment when I submit the PR. but now it's ok.

---

okay, thanks for the clarification.